include/dungeon_map_access.h

```diff
@@ -5,10 +5,10 @@
 
 struct unkStruct_202F190
 {
-    struct MapTile* unk0[6];
+    struct Tile* unk0[6];
 };
 
-struct MapTile* GetMapTile_1(s32 x, s32 y);
-struct MapTile* GetMapTile_2(s32 x, s32 y);
+struct Tile* GetTile(s32 x, s32 y);
+struct Tile* GetTileSafe(s32 x, s32 y);
```

Comment:
What is meant by Safe here?

Reply:
https://github.com/UsernameFodder/pmdsky-debug/blob/master/symbols/overlay29.yml#L3796

GetTile():
`Get the tile at some position. If the coordinates are out of bounds, returns a default tile.`
GetTileSafe():
`Get the tile at some position. If the coordinates are out of bounds, returns a pointer to a copy of the default tile.`